massa-client/src/cmds.rs

```diff
@@ -235,14 +235,14 @@ pub enum Command {
 
```

Comment:
* line 222: "props(args = "SenderAddress PathToBytecode MaxGas GasPrice Fee",)," :  gas price should be removed

* line 230: "SenderAddress TargetAddress FunctionName Parameter MaxGas GasPrice Coins Fee" => gas price should be removed

Reply:
Done, thanks!